bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>i think there could be a bit more explaination before this code block. e.g. emphasizing that here we need to define a few agent tools through natural language descriptions, and the tools are what the "agent service" blocks in the top graph is about. otherwise the user may have a hard time understanding how come a few description strings help in this story.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463984'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>now looks much better.</p>